From 332c30282cc89e7a2d773a5835ddbd646dece6ff Mon Sep 17 00:00:00 2001
From: Mario Alfredo Carrillo Arevalo <mario.alfredo.c.arevalo@intel.com>
Date: Tue, 8 Nov 2016 17:45:26 -0600
Subject: [PATCH] disable gst-segementation plugin

OpenCV pulgins for gstreamer cannot be built,
because one of them asks for a dependence that
so far has not been included in Clear Linux.
This dependence is an extra module called bgsegm,
and it is part of "opencv_contrib" project:

https://github.com/opencv/opencv_contrib

opencv_contrib project is intended for development
of so-called "extra" modules, contributed functionality.

Signed-off-by: Mario Alfredo Carrillo Arevalo <mario.alfredo.c.arevalo@intel.com>
---
 ext/opencv/Makefile.am   | 1 -
 ext/opencv/gstopencv.cpp | 4 ----
 2 files changed, 5 deletions(-)

diff --git a/ext/opencv/Makefile.am b/ext/opencv/Makefile.am
index 360d4ab..9ba8424 100644
--- a/ext/opencv/Makefile.am
+++ b/ext/opencv/Makefile.am
@@ -20,7 +20,6 @@ libgstopencv_la_SOURCES = gstopencv.cpp \
 			gstskindetect.cpp \
 			gstretinex.cpp \
 			gstfacedetect.cpp \
-			gstsegmentation.cpp \
 			gstgrabcut.cpp \
 			gstdisparity.cpp \
 			motioncells_wrapper.cpp \
diff --git a/ext/opencv/gstopencv.cpp b/ext/opencv/gstopencv.cpp
index 8f50cd8..209078d 100644
--- a/ext/opencv/gstopencv.cpp
+++ b/ext/opencv/gstopencv.cpp
@@ -38,7 +38,6 @@
 #include "gsthanddetect.h"
 #include "gstskindetect.h"
 #include "gstretinex.h"
-#include "gstsegmentation.h"
 #include "gstgrabcut.h"
 #include "gstdisparity.h"
 
@@ -90,9 +89,6 @@ plugin_init (GstPlugin * plugin)
   if (!gst_retinex_plugin_init (plugin))
     return FALSE;
 
-  if (!gst_segmentation_plugin_init (plugin))
-    return FALSE;
-
   if (!gst_grabcut_plugin_init (plugin))
     return FALSE;
 
-- 
2.10.0.372.g6fe1b14

